circleguard/settings.py

```diff
@@ -130,6 +130,7 @@
         "message_loading_replays":         "[{ts:%X}] Loading {num_unloaded} of {num_total} replays ({num_previously_loaded} replays previously loaded)",
         "message_ratelimited":             "[{ts:%X}] Ratelimited, waiting for {s} seconds",
         "message_starting_investigation":  "[{ts:%X}] Running {check_type} check",
+        "message_starting_investigation_visualization":  "[{ts:%X}] Running visualization check, you can visualize from the Results tab when finished",
```

Comment:
Are you sure that we want to call it a visualization "Check"? How about "Block" or "Object"?

Reply:
check isn't the best but those are both worse. What if we just dropped it entirely and said "running visualization"

Reply:
well then I would still prefer visualization "Check" since "running visualization" is a bit misleading. (It isn't running the visualization, it's adding it to the result tab so you can visualize it later)

Reply:
yeah that's true, I realized that as well. I think visualization check is fine